shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -91,6 +91,8 @@ class FlutterPlatformViewsController {
   GrContext* overlays_gr_context_;
   SkISize frame_size_;
 
+  std::unordered_set<int64_t> views_waiting_to_dispose_;
```

Comment:
nit: views_to_dispose_
Please also add a comment explaining that method channel dispose calls adds the view to this set to be disposed on the next frame.

Reply:
Done